src/CMake/CheckMinimumCompilerVersion.cmake

```diff
@@ -45,8 +45,11 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Intel" AND
   message(FATAL_ERROR "Intel C++ (ICC) 14.0 or later is required.")
 endif ()
 
-set(CMAKE_CXX_STANDARD 11)
+#-----------------------------------------------------------------------------
+# VTKm requires C++14. Turning off extensions for maximum compatability.
```

Comment:
So, I am curious about this...Does VTKm require C++14 to build or does any caller of the VTKm interface also require C++14?

Reply:
The C++14 usage bleeds through the VTKm interface, so VisIt needs C++14.